### PR TITLE
added coopa-troopa

### PR DESCRIPTION
**Pull request by CooperSawyer1**

### CODE REVIEW

**Comment by RudeChowder** [comment]
seems legit

File: README.md [hunk 1]
```diff
@@ -3,8 +3,12 @@
 This is a repository to test making pull requests. It has been contributed to by:
 
 Gillian Wenhold
+
+Coopa- troopa
+
 Cooper's Grey Hair
```
Comment (gillianwenhold) [inline README.md:9]: Why are you referenced twice?

**Comment by RobGentile17** [comment]
Not good at all you fail

**Review by RudeChowder — APPROVED** [review]
A little sus, but I'll let it slide

**Review by gillianwenhold — CHANGES_REQUESTED** [review]
You're in there twice

File: README.md (hunk 1 above)
Comment (CooperSawyer1) [reply README.md:9]: for posterity

Comment (gillianwenhold) [reply README.md:9]: Oh okay

**Review by gillianwenhold — APPROVED** [review]
This is fine

**Pull request merged**